kong/plugins/response-ratelimiting/schema.lua

```diff
@@ -207,6 +239,10 @@ return {
           } },
           { redis_database = {
             type = "integer",
+            include_in_output = true,
+            translate_backwards = function(instance)
+              return instance.redis.database
+            end,
```

Comment:
I find the repetition of these clauses bothersome, and I also don't like `translate_backwards` to be a function, as that creates flexibility which is not needed and could lead to difficult to remove code later on.  Also, do we need `include_in_output`?  Can't the functionality be triggered by the presence of `translate_backwards?

```suggestion
            translate_backwards = {'redis', 'database'}
```

Reply:
All functions that we add make it more difficult for the Koko team to replicate our schemas on their side, as well. They'd rather have things as declarative as possible (no functions, just fields)

Reply:
I removed `include_in_output` and changed the structure of `translate_backwards` to be a table of keys.